Update the Readme (#1)
DevExpress-Examples/asp-net-mvc-spell-checker-customize-built-in-dialogs: Readme.md

# ASP.NET MVC SpellChecker - Customize built-in dialogs

> This example customizes built-in spell check dialogs in [ASP.NET MVC Spell Checker](https://docs.devexpress.com/AspNetMvc/DevExpress.Web.Mvc.MVCxSpellChecker).

Reviewer: ```suggestion
This example customizes built-in spell check dialogs in the [ASP.NET MVC Spell Checker](https://docs.devexpress.com/AspNetMvc/DevExpress.Web.Mvc.MVCxSpellChecker).

```

## Implementation Details

The DevExpress ASP.NET MVC Spell Checker ships with two built-in forms:

> 1. **SpellCheckForm** allows users to accept/reject spelling suggestions and update a custom dictionary with new words.

Reviewer: Which custom dictionary? Can you specify what you are referring to?

Author: ```suggestion
1. **SpellCheckForm** allows users to accept/reject spelling suggestions and update the dictionary with new words.

```
